types/account.ts

```diff
@@ -292,15 +392,27 @@ export interface AccountDepositProduct {
     }
 }
 
-export type CloseReason = "ByCustomer" | "Fraud"
+export type CloseAccountType = "depositAccountClose" | "creditAccountClose"
 
 export interface CloseAccountRequest {
     accountId: string
 
     data: {
-        type: "accountClose"
+        type: CloseAccountType
         attributes: {
-            reason: CloseReason
+            /**
+             * The reason for closing the account.
+             * For Deposit Account Either ByCustomer or Fraud.
+             * For Credit Account Either ByCustomer, Fraud or Overdue.
+             * If not specified, will default to ByCustomer.
+             */
+            reason: CloseReason | "Overdue"
```

Comment:
How do you feel about making `CloseReason | "Overdue"` as CreditAccountCloseReason?
feels kinda shaky if we need to repeat it
(I think I understand the backward-compatibility dillema)